### PR TITLE
Bug fix/windows username not recognized

### PR DESCRIPTION
**Pull request by eyalk007**
this branch solves the bug for windows users who have spaces in their username.
these users can now use the plugin normally.

### CODE REVIEW

File: host/windows/runcli.bat [hunk 1]
```diff
@@ -1,17 +1,19 @@
 @ECHO OFF
 
 :: Runs JFrog CLI with the given arguments. The JFrog CLI's home directory is the extension's home directory.
-
-set HOME_DIR=%USERPROFILE%\.jfrog-docker-desktop-extension
-set LOGS_DIR=%HOME_DIR%\logs
-
-set JFROG_CLI_HOME_DIR=%HOME_DIR%
+set "HOME_DIR=%USERPROFILE%\.jfrog-docker-desktop-extension"
```
Comment (hadarshjfrog) [inline host/windows/runcli.bat:4]: I think thq quotes should be on the variable itself - not the entire SET:
```suggestion
set HOME_DIR="%USERPROFILE%\.jfrog-docker-desktop-extension"
```

File: host/windows/runcli.bat [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+set "LOGS_DIR=%HOME_DIR%\logs"
+set "JFROG_CLI_HOME_DIR=%HOME_DIR%"
 set JFROG_CLI_USER_AGENT=jfrog-docker-extension
 set JFROG_CLI_LOG_LEVEL=DEBUG
 set CI=true
 
-if not exist %LOGS_DIR% mkdir %LOGS_DIR%
+:: Create logs directory if it doesn't exist, handling paths with spaces.
+if not exist "%LOGS_DIR%" mkdir "%LOGS_DIR%"
```
Comment (hadarshjfrog) [inline host/windows/runcli.bat:12]: isn't the quotes here redundent? Either here or in the definitions themselves - don't think we need both...

**Review by hadarshjfrog — CHANGES_REQUESTED** [review]

File: host/windows/runcli.bat (hunk 1 above)
Comment (eyalk007) [reply host/windows/runcli.bat:4]: okay. will check when I have the chance if this change works

File: host/windows/runcli.bat (hunk 2 above)
Comment (eyalk007) [reply host/windows/runcli.bat:12]: I have a feeling they are. because when I echoed %CONF_FILE% I needed them.
I will try without and update accordingly.

Comment (eyalk007) [reply host/windows/runcli.bat:12]: nope didnt work

File: host/windows/runcli.bat (hunk 1 above)
Comment (eyalk007) [reply host/windows/runcli.bat:4]: also no sadly 🥲

**Pull request merged**